src/Klarna.Checkout/KlarnaCheckoutService.cs

```diff
@@ -337,23 +346,32 @@ public virtual ShippingOptionUpdateResponse UpdateShippingMethod(ICart cart, Shi
                 OrderAmount = AmountHelper.GetAmount(totals.Total),
                 OrderTaxAmount = AmountHelper.GetAmount(totals.TaxTotal),
                 OrderLines = GetOrderLines(cart, totals, configuration.SendProductAndImageUrl),
-                PurchaseCurrency = cart.Currency.CurrencyCode
+                PurchaseCurrency = cart.Currency.CurrencyCode,
+                ShippingOptions = configuration.ShippingOptionsInIFrame ? GetShippingOptions(cart, cart.Currency, ContentLanguage.PreferredCulture) : Enumerable.Empty<ShippingOption>(),
```

Comment:
Could you please split into separate rows? It is too long.
```suggestion
                ShippingOptions = configuration.ShippingOptionsInIFrame
                    ? GetShippingOptions(cart, cart.Currency, ContentLanguage.PreferredCulture)
                    : Enumerable.Empty<ShippingOption>(),
```